iroh-bytes/src/provider.rs

```diff
@@ -466,6 +524,10 @@ async fn handle_stream<E: EventSender>(
         }
     };
 
+    authorization_handler
+        .authorize(db.clone(), request.token().cloned(), &request)
```

Comment:
Sorry I missed this until now, given that we have to clone this here, the token should not be `Vec<u8>` but rather `Bytes`, so that clones are cheap reference counts.